packages/eui/src/services/color/contrast.ts

```diff
@@ -104,3 +107,19 @@ export const makeDisabledContrastColor =
         }
   ) =>
     makeHighContrastColor(color, ratio)(themeOrBackground);
+
+    export const warnIfContrastBelowMin = (
+  textColor: string,
+  backgroundColor: string,
+  min = wcagContrastMin
+) => {
+  const ratio = getColorContrast(textColor, backgroundColor);
+  if (ratio < min) {
+    // eslint-disable-next-line no-console
+    console.warn(
+      `Warning: ${backgroundColor} background with ${textColor} text has a low contrast of ${ratio.toFixed(
+        2
+      )}. Should be above ${min}.`
+    );
+  }
+};
```

Comment:
and here it's complaining about no empty line in the end:

```suggestion
};

```